mir-module/src/main/resources/xsl/user-profile/user-profile-attributes-defaults.xsl

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" exclude-result-prefixes="xsl">
+
+  <xsl:import href="resource:xsl/user-profile/user-profile-attribute-utils.xsl"/>
+  <xsl:import href="xslImport:userProfileAttributes:user-profile/user-profile-attributes-defaults.xsl"/>
+
+  <xsl:template match="user" mode="user-attributes">
+    <xsl:apply-imports/>
+
+    <xsl:if test="$fullDetails">
```

Comment:
As a normal user, you no longer see any attributes. The condition must have accidentally been added and should be removed.